pom.xml

```diff
@@ -81,33 +81,6 @@
       * io.opentelemetry.instrumentation:opentelemetry-resources
       * io.opentelemetry.contrib:opentelemetry-gcp-resources
       -->
-      <dependency>
-        <groupId>io.opentelemetry.semconv</groupId>
-        <artifactId>opentelemetry-semconv</artifactId>
-        <version>1.27.0-alpha</version>
-      </dependency>
-      <dependency>
-        <groupId>com.google.cloud.opentelemetry</groupId>
-        <artifactId>detector-resources</artifactId>
-        <version>0.27.0-alpha</version>
-        <exclusions>
-          <exclusion>
-            <groupId>io.opentelemetry.semconv</groupId>
-            <artifactId>opentelemetry-semconv</artifactId>
-          </exclusion>
-        </exclusions>
-      </dependency>
-      <dependency>
-        <groupId>io.opentelemetry.instrumentation</groupId>
-        <artifactId>opentelemetry-resources</artifactId>
-        <version>2.6.0-alpha</version>
-        <exclusions>
-          <exclusion>
-            <groupId>io.opentelemetry.semconv</groupId>
-            <artifactId>opentelemetry-semconv</artifactId>
-          </exclusion>
-        </exclusions>
-      </dependency>
       <dependency>
         <groupId>io.opentelemetry.contrib</groupId>
         <artifactId>opentelemetry-gcp-resources</artifactId>
```

Comment:
Can we remove the exclusion from this as well? Now that the semceonv is managed by the bom it's version will take precedence.